### PR TITLE
QOL-8104 Cleanup

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Move S3 robots block into prod area since non-prod already disallows all
- Sync CI config with other repos

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -25,3 +25,16 @@ jobs:
       - name: Test
         run: .circleci/test.sh
         timeout-minutes: 15
+
+      - name: Retrieve screenshots
+        if: failure()
+        run: .circleci/process-artifacts.sh
```
Comment (duttonw) [inline .github/workflows/test.yml:31]: is process-artifacts.sh the correct name for this? should it not be renamed to match what it does?

File: requirements-dev.txt [hunk 2]
```diff
@@ -4,9 +4,9 @@ Appium-Python-Client<=0.52
 flake8==3.8.3
 nose==1.3.7
 splinter>=0.13.0
--e git+https://github.com/qld-gov-au/ckanext-datarequests@2.1.1-qgov#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@3.0.1#egg=ckanext-data_qld
--e git+https://github.com/qld-gov-au/ckan-ex-qgov@3.1.0#egg=ckan-ex-qgov
+-e git+https://github.com/qld-gov-au/ckanext-datarequests@develop#egg=ckanext-datarequests
```
Comment (duttonw) [inline requirements-dev.txt:7]: so we are back to not needing to lock these again?

**Review by duttonw — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (ThrawnCA) [reply .github/workflows/test.yml:31]: It's for copying screenshots out of the container and into the CI workspace. I think it's fine.

It's also consistent with a dozen other repos at this point.

File: requirements-dev.txt (hunk 2 above)
Comment (ThrawnCA) [reply requirements-dev.txt:7]: Good point, we did change that to fixed versions deliberately. I'll change back but update them.

**Pull request merged**